packages/terra-table/src/subcomponents/_Cell.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import styles from './Cell.module.scss';
+import { styleFromWidth } from './utils';
+import widthShape from '../proptypes/widthShape';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child content to be displayed for the row cell.
+   */
+  children: PropTypes.node,
+  /**
+   * Data to attach with the link role of the cell.
```

Comment:
I'm not sure I understand this prop name/description. Below it appears by passing this prop, the cell it's loses tab-ability and lets the content mange it's accessibility?

the name `disclosureData` makes me think clicking it would enable some sort of integration with the disclosure manager API